tests/pytests/functional/formulas/test_salt.py

```diff
@@ -13,18 +13,41 @@ def _formula(saltstack_formula):
 
 @pytest.fixture(scope="module")
 def modules(loaders, _formula):
+    loaders.opts["file_roots"]["base"].append(
+        str(_formula.state_tree_path / f"{_formula.name}-{_formula.tag}")
+    )
     return loaders.modules
 
 
 @pytest.mark.skip_on_windows
 @pytest.mark.destructive_test
+# These installation issues need to be resolved in the Salt formula, and then
+# we can update the tag in here and remove the skips
+@pytest.mark.skipif(
+    'grains["osfullname"] in ("AlmaLinux", "Amazon Linux")',
+    reason="No salt packages available for this distrubition",
+)
+@pytest.mark.skipif(
+    'grains["os"] == "CentOS"',
+    reason="No salt packages available for this distrubition",
+)
+@pytest.mark.skipif(
+    'grains["os_family"] == "Arch"',
+    reason="Outdated archlinux-keyring package will cause failed package installs",
+)
 def test_salt_formula(modules):
     # Master Formula
     ret = modules.state.sls("salt.master")
+    assert not ret.errors
+    # asserting not ret.failed will cause a False positive if ret.failed is None
+    assert ret.failed is not True
```

Comment:
```suggestion
    assert ret.failed is False
```
Rule of thumb, if the expected value is a Boolean, always assert against the expected `is True` or `is False`

Reply:
This has been fixed.